### DISCUSSION TITLE
Using Github logo for Identity provider integration

### DISCUSSION THREAD

**Discussion by chouclee** [community/community#63980] [Other Feature Feedback, Questions, & Ideas]
Our app offers identity provider integration with Github IDP (also other IDPs like Google, openID etc) so users can sign in using Github. Does anyone know if it's ok for us to use Github logo to indicate this is an integration with Github?

I have filed support ticket to Github but never get any responses so far.

**Comment by FreeZe5K** [comment]
Have you tried referencing the [Logos and Usage](https://github.com/logos) guidelines?

In general:

1) Don't use their logos or name in a way that suggests any kind of official affiliation or partnership unless you have a specific agreement.
2) Respect their color palette and design specifics.
3) Always provide significant spacing around the logo.

While integrating with IDPs, it's common to use the respective logos to indicate the option, but it should be clear that it's just an integration and not an endorsement or partnership. As always, to ensure full compliance, I'd recommend waiting for a direct response from GitHub Support or consult with a legal expert who specializes in intellectual property or brand use. The support ticket is a good start, but definitely follow the link I added for additional clarification while you wait.
